README.md

```diff
@@ -4,4 +4,28 @@ Launch Program.
 
 Expected output: "Hello World!"
 
-Enjoy
+## FUNctions
+
+A C++ function **definition** consists of a function header and a function body. A function **definition** tells the compiler about a function's name, return type, and parameters. A function definition provides the actual body of the function.
+
+Here are all the parts of a function −
+
+_Return Type_ − A function may return a value. The return_type is the data type of the value the function returns. Some functions perform the desired operations without returning a value. In this case, the return_type is the keyword void.
+
+_Function Name_ − This is the actual name of the function. The function name and the parameter list together constitute the function **signature**.
+
+_Parameters_ − A parameter is like a placeholder. When a function is invoked, you pass a value to the parameter. This value is referred to as actual parameter or argument. The parameter list refers to the type, order, and number of the parameters of a function. Parameters are optional; that is, a function may contain no parameters.
+
```

Comment:
Correct. I would also add in here "parameters are also called function arguments."

Reply:
I would also say, instead of "you pass a value to the parameter", this instead:

"When a function is invoked, you can pass things in to the function through its parameters/arguments."

The parameters aren't what we pass things to; we pass things to the functions themselves, and the parameters/arguments are the mechanisms by which we do so!